beacon_node/beacon_chain/src/data_availability_checker/overflow_lru_cache.rs

```diff
@@ -0,0 +1,1653 @@
+use crate::beacon_chain::BeaconStore;
+use crate::blob_verification::KzgVerifiedBlob;
+use crate::block_verification::{AvailabilityPendingExecutedBlock, AvailableExecutedBlock};
+use crate::data_availability_checker::{Availability, AvailabilityCheckError};
+use crate::store::{DBColumn, KeyValueStore};
+use crate::BeaconChainTypes;
+use lru::LruCache;
+use parking_lot::{Mutex, RwLock, RwLockUpgradableReadGuard, RwLockWriteGuard};
+use ssz::{Decode, Encode};
+use ssz_derive::{Decode, Encode};
+use ssz_types::FixedVector;
+use std::{collections::HashSet, sync::Arc};
+use types::blob_sidecar::BlobIdentifier;
+use types::{BlobSidecar, Epoch, EthSpec, Hash256};
+
+/// Caches partially available blobs and execution verified blocks corresponding
+/// to a given `block_hash` that are received over gossip.
+///
+/// The blobs are all gossip and kzg verified.
+/// The block has completed all verifications except the availability check.
+#[derive(Encode, Decode, Clone)]
+pub struct PendingComponents<T: EthSpec> {
+    /// We use a `BTreeMap` here to maintain the order of `BlobSidecar`s based on index.
```

Comment:
I think this was missed in a prior PR but this comment is outdated
```suggestion
    /// We use a `BTreeMap` here to maintain the order of `BlobSidecar`s based on index.
```

Reply:
I don't see a change in what you proposed. I think you meant to delete this? I'll delete the comment.